utils/metabot.js

```diff
@@ -16,18 +16,24 @@ function* handleMessageCreate(event) {
     let args        = _.tail(tokens)
     let command     = commands[commandName]
 
-    if (command) {
-
-      try {
-        let result = yield command(args)
-        event.message.channel.sendMessage(result)
-      }
-      catch (err) {
-        event.message.channel.sendMessage(
-          METABOT.COMMAND_ERROR_MESSAGE + ': ' + err
-        )
-      }
+    if (!command) {
+      return
+    }
+
+    try {
+
+      let result = yield command({
+        args,
+        message : event.message
+      })
```

Comment:
Actually, looking at it, this is the event handler for `MESSAGE_CREATE`, so it makes sense that only message would be passed through here.